templates/previewDebugUrl.twig

```diff
@@ -0,0 +1,7 @@
+<br>
+<div class="preview-debug-url-div">
+    <input type="url" id="previewDebugUrl" class="browser-default" placeholder="https://yourdebugsite.com" value="{{ debugSiteUrl|e('html_attr') }}">
+    <button type="button" class="btn btn-small" onclick='tagManagerHelper.changeDebugUrl({{ idcontainer }},"{{ debugSiteUrl|e('html_attr') }}")'>
```

Comment:
fyi noticed when entering eg `foo` and then press debug then it will open `https://yourdomain.com/foo`. That can be actually good I suppose. Similar when entering `?er454534` into the text field etc.